### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - Aude Laydu et Leonard Cseres

### DISCUSSION THREAD

**Discussion by leonardcser** [heig-vd-dai-course/heig-vd-dai-course#482] [Show and tell]
Application Multicast UDC, qui permet à des clients de s'affronter dans une course de dactylographie.

Le serveur gère uniquement une partie, que plusieurs joueurs peuvent rejoindre.

Inspiration:
https://play.typeracer.com

**Comment by ludelafo** [comment]
OK pour nous, mais je pense que du multicast UDP est plus adapté que du multicast UDC.. :)

**Reply by eau2** [reply]
Here's the link to our repository : https://github.com/HEIGVD-S3-DAI/fastpaws 
and the hash of the commit we want to submit : dd96100c8634f82b87130d67d040310f4cb3d59a
